### PR TITLE
ci: Cypress test vercel preview link 

### PR DESCRIPTION
**Pull request by chef-jojo**
Fix nft collection test, after https://github.com/pancakeswap/pancake-frontend/pull/3244
Cypress is unstable when accessing remote url. 
Add retry and ignore info test temporally

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -5,5 +5,9 @@
   "supportFile": "cypress/support/index.js",
   "video": false,
   "defaultCommandTimeout": 10000,
-  "chromeWebSecurity": false
+  "chromeWebSecurity": false,
+  "pageLoadTimeout": 10000,
+  "retries": {
+    "runMode": 2
```
Comment (chef-jojo) [inline cypress.json:11]:
```suggestion
    "runMode": 3
```

File: cypress.json [hunk 2]
```diff
@@ -5,5 +5,11 @@
   "supportFile": "cypress/support/index.js",
   "video": false,
   "defaultCommandTimeout": 10000,
-  "chromeWebSecurity": false
+  "chromeWebSecurity": false,
+  "pageLoadTimeout": 10000,
+  "blockHosts": ["vitals.vercel-insights.com", "*sentry.io", "*googletagmanager.com"],
+  "ignoreTestFiles": ["info.test.ts"],
```
Comment (chef-jojo) [inline cypress.json:11]: skip info test. e2e becomes fails alarm we literally ignore it and may miss some major error  🤷‍♂️

Comment (chefilip) [reply cypress.json:11]: Could we add a comment on this line so other devs could know why we ignore it? Thanks

Comment (chef-jojo) [reply cypress.json:11]: it's json. That's why I comment on this. I'll remove this once we know info is back to normal

**Pull request merged**